src/puppetlabs/puppetdb/utils.clj

```diff
@@ -237,6 +237,14 @@
    :prefix "/pdb/cmd"
    :version (or version :v1)})
 
+(defn pdb-meta-base-url
+  [host port & [version]]
+  {:protocol "http"
+   :host host
+   :port port
+   :prefix "/pdb/meta"
+   :version (or version :v1)})
```

Comment:
yeah, I just noticed that. We probably shouldn't, but if it'd be better as a separate change I wouldn't mind keeping it for now.